tests/integration/helpers/factories/db/invenio_records.py

```diff
@@ -81,25 +81,34 @@ def create_from_kwargs(cls, index=True, has_pid=True, **kwargs):
         instance.record_metadata = super(TestRecordMetadata, cls)\
                 .create_from_kwargs(updated_kwargs)
 
-        if index:
+        if index == 'records-hep':
```

Comment:
I'd also pass the ``doc_type`` as a keyword, so we don't have any index restriction since we want to make it as abstract as possible

Reply:
we could also add ``@lassmethods`` for our two main cases, ``create_literature`` and ``create_data`` so we can use them easily

Reply:
class methods seems like a nice idea.

Reply:
Coming to think of this again, do we really need separate methods for that? With the current configuration (latest commit), we can essentially use 1 required(if we want to index the record in ES) and 1 optional arguments to create any type of record: index_name and pid_type.

Reply:
it'd be cleaner for the tests, just to call what you need without having "hard coded" strings, if you think that it is out of the US's scope, we can leave it as is.

Reply:
I suppose I'll leave it like this for now.